pkg/stub/handler.go

```diff
@@ -58,18 +61,18 @@ func NewSamplesOperatorHandler(kubeconfig *restclient.Config, cachestore control
 		return nil, err
 	}
 	crdWrapper.client = client.Configs()
-	crdWrapper.store = cachestore
+	crdWrapper.store = configObjCacheStore
 
 	h.crdwrapper = crdWrapper
 
-	h.crdstore = cachestore
+	h.crdstore = configObjCacheStore
 
 	h.Fileimagegetter = &DefaultImageStreamFromFileGetter{}
 	h.Filetemplategetter = &DefaultTemplateFromFileGetter{}
 	h.Filefinder = &DefaultResourceFileLister{}
 
-	h.imageclientwrapper = &defaultImageStreamClientWrapper{h: h}
-	h.templateclientwrapper = &defaultTemplateClientWrapper{h: h}
+	h.imageclientwrapper = &defaultImageStreamClientWrapper{h: h, store: streamCacheStore}
+	h.templateclientwrapper = &defaultTemplateClientWrapper{h: h, store: streamCacheStore}
```

Comment:
this is also a problem ... gonna address this first, see where we land afterward